etc/acl_manager.cfg

```diff
@@ -1,6 +1,8 @@
 [global]
 # Plugin address
 PluginAddress = controller
+# Local address to bind to; IP address or "*" (the default).
```

Comment:
Doesn't match the comment used in the other config file, despite being identical fields.